pages/docs/community/styleguide/formatting.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Formatting
+description: This guide illustrates the standards for formatting and writing our documentation.
+weight: 11
```

Comment:
@alequetzalli Do verify that it follows the weight structure and it's showing the correct sequence of the pages inside docs.